assets/keymaps/default-linux.json

```diff
@@ -68,14 +68,10 @@
         "end": "editor::MoveToEndOfLine",
         "left": "editor::MoveLeft",
         "right": "editor::MoveRight",
-        "ctrl-p": "editor::MoveUp",
-        "ctrl-n": "editor::MoveDown",
-        "ctrl-b": "editor::MoveLeft",
-        "ctrl-f": "editor::MoveRight",
```

Comment:
I like these one. But if we remove them, we should probably also remove their equivalents with selection (with `shift`).

Reply:
Yeah, why should these be removed?

Reply:
1. `ctrl-n` conflicts with the shortcut to open a new tab
2. They seem to be MacOS specific and AFAIK they are not used in any other Linux app

Reply:
Ahh, I see. Makes sense.

Reply:
Those are part of the emacs keybindings, and are occasionally present in linux. It's fine to remove but then probably we should remove all of them. The `b` and `f` ones (backward and forward) are: `ctrl-_`, `alt-_`, `ctrl-shift-_`, `alt-shift-_`. But there's also a few more of them like `ctrl-(shift)-a`, `ctrl-(shift)-e`, `ctrl-u`, `ctrl-d`. Might be forgetting more of them.

Splitting them in a different file might be nice, in case we make an option to enable them in the future.